src/main/jbake/content/docs/user/user-guide.adoc

```diff
@@ -10,14 +10,17 @@ Thomas Heute
 
 toc::[]
 
-include::metrics.adoc[]
+include::metrics.adocPart[]
 
-include::alerts.adoc[]
+// this now works
+//include::https://cdn.rawgit.com/hawkular/hawkular-metrics/master/api/metrics-api-jaxrs/README.adoc[]
```

Comment:
@theute If your idea was to include the remote `README.adoc` files, this is now possible (`allow-uri-read=true` in `jbake.properties` does the work)